src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -26,22 +29,31 @@ export const monoPlatform: Platform = {
   },
 
   findMethod: function findMethod(assemblyName: string, namespace: string, className: string, methodName: string): MethodHandle {
-    // TODO: Cache the assembly_load outputs?
-    const assemblyHandle = assembly_load(assemblyName);
+
+    var assemblyHandle: number | undefined = registeredAssemblies[assemblyName];
```

Comment:
Could we use `let` rather than `var`? In general `var` can always be replaced by `let` or `const`, depending on whether the value needs to change.

Reply:
Also below

Reply:
const failed as the value changes; sorry I'm a noob to typescript/javascript and learning as I implement ;) I will change the var to let